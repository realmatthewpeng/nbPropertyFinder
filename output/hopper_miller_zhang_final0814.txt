Total number of lines found: 48

data['OrderedColor'] = data.Color.apply(reorder)
data['TopBreed'] = data.Breed.apply(top_breed)
data['MixedBreed'] = data.Breed.apply(mixed_breed)
data['Female'] = data.SexuponOutcome.apply(female)
data['Intact'] = data.SexuponOutcome.apply(intact)
data['OutcomeSubtype'] = data['OutcomeSubtype'].fillna('')
data['OrderedBreed'] = data.Breed.apply(reorder)
data['BreedRank'] = data.Breed.apply(breed_rank)
data['Named'] = data.Name.apply(naming)
breeds['Breed'] = breeds['Breed'].str.strip()
data['PitBull'] = data.Breed.apply(pit_bull)
data['ConvertedAge']=data['AgeuponOutcome'].apply(ageConvert)
if i.find(" ") >= 0: i = i.replace(" ", "-")
ranks.append(int(breeds.loc[breeds['Breed'] == word]['2007']))
data['BlackCat'] = data.Color.apply(black_cat)
if pd.isnull(i): return 'Unnamed'
data['AgeuponOutcome'].fillna('', inplace = True)
if i.find("/") >= 0: i = i.replace("/", " ")
return np.mean(ranks)
data['Female'] = 'Female' in data['SexuponOutcome']
X = data2[predictors]
data["ConvertedAge"] = data.groupby("AnimalType").transform(lambda x: x.fillna(x.median()))
data2 = pd.concat([data[target], data[continuous], pd.get_dummies(data[discrete])], axis=1)
y = data2[[target]]
data[continuous].describe().T
animal_type = data['AnimalType'].value_counts()
outcome_type = data['OutcomeType'].value_counts()
print("Unique Breeds" , (data.OrderedBreed.value_counts() > 0).sum())
OrderedBreed = data.OrderedBreed.unique()
plt.xticks(range(len(corr.columns)), corr.columns,rotation='vertical');
plt.yticks(range(len(corr.columns)), corr.columns);
X_cat = data_cat[pred['cat']]
y_cat = data_cat[[target]]
data_cat = data[data['AnimalType'] == 'Cat']
data_dog = data[data['AnimalType'] == 'Dog']
y_dog = data_dog[[target]]
X_dog = data_dog[pred['dog']]
data = X[self.cols]
cm = cm.astype('float') / cm.sum(axis=1)[:, np.newaxis]
thresh = cm.max() / 2.
X_dev_feature = X_dev_feature.drop('Intact_Unknown', 1)
X_dev_feature = X_dev_feature.drop('Female_Unknown', 1)
X_train_feature = X_train_feature.drop('Intact_Unknown', 1)
X_train_feature = X_train_feature.drop('Female_Unknown', 1)
X_dev_dog = X_dev_feature.drop('AnimalType_Dog', 1)
X_train_dog = X_train_feature.drop('AnimalType_Dog', 1)
X_dev_cat = X_dev_feature.drop('AnimalType_Cat', 1)
X_train_cat = X_train_feature.drop('AnimalType_Cat', 1)

Incorrect Lines:

if i.find(" ") >= 0: i = i.replace(" ", "-")
ranks.append(int(breeds.loc[breeds['Breed'] == word]['2007']))
if pd.isnull(i): return 'Unnamed'
if i.find("/") >= 0: i = i.replace("/", " ")
return np.mean(ranks)
X = data2[predictors]
y = data2[[target]]
OrderedBreed = data.OrderedBreed.unique()
plt.xticks(range(len(corr.columns)), corr.columns,rotation='vertical');
plt.yticks(range(len(corr.columns)), corr.columns);
data = X[self.cols]
thresh = cm.max() / 2.

Precision = 0.75
Recall = 0.8

ss.fit(X_train[continuous])   # Compute mean and std of training data
X_train[continuous] = ss.transform(X_train[continuous])  # Use that mean and std to normalize columns of training data
X_dev[continuous] = ss.transform(X_dev[continuous])
X_train_feature = X_train
X_dev_feature = X_dev
X_train_dog = X_train_feature
X_dev_dog = X_dev_feature
X_train_cat = X_train_feature
X_dev_cat = X_dev_feature
