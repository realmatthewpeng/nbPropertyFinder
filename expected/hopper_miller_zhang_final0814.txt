breeds['Breed'] = breeds['Breed'].str.strip()
data['OutcomeSubtype'] = data['OutcomeSubtype'].fillna('')
data['Female'] = 'Female' in data['SexuponOutcome']
data['AgeuponOutcome'].fillna('', inplace = True)
data['ConvertedAge']=data['AgeuponOutcome'].apply(ageConvert)
data['Female'] = data.SexuponOutcome.apply(female)
data['Intact'] = data.SexuponOutcome.apply(intact)
data['MixedBreed'] = data.Breed.apply(mixed_breed)
data['TopBreed'] = data.Breed.apply(top_breed)
data['BreedRank'] = data.Breed.apply(breed_rank)
data['PitBull'] = data.Breed.apply(pit_bull)
data['BlackCat'] = data.Color.apply(black_cat)
data['Named'] = data.Name.apply(naming)
data['OrderedColor'] = data.Color.apply(reorder)
data['OrderedBreed'] = data.Breed.apply(reorder)
data["ConvertedAge"] = data.groupby("AnimalType").transform(lambda x: x.fillna(x.median()))
data[continuous].describe().T
data2 = pd.concat([data[target], data[continuous], pd.get_dummies(data[discrete])], axis=1)
ss.fit(X_train[continuous])   # Compute mean and std of training data
X_train[continuous] = ss.transform(X_train[continuous])  # Use that mean and std to normalize columns of training data
X_dev[continuous] = ss.transform(X_dev[continuous])
animal_type = data['AnimalType'].value_counts()
outcome_type = data['OutcomeType'].value_counts()
print("Unique Breeds" , (data.OrderedBreed.value_counts() > 0).sum())
data_dog = data[data['AnimalType'] == 'Dog']
data_cat = data[data['AnimalType'] == 'Cat']
X_dog = data_dog[pred['dog']]
X_cat = data_cat[pred['cat']]
y_dog = data_dog[[target]]
y_cat = data_cat[[target]]
cm = cm.astype('float') / cm.sum(axis=1)[:, np.newaxis]
X_train_feature = X_train
X_train_feature = X_train_feature.drop('Intact_Unknown', 1)
X_train_feature = X_train_feature.drop('Female_Unknown', 1)
X_dev_feature = X_dev
X_dev_feature = X_dev_feature.drop('Intact_Unknown', 1)
X_dev_feature = X_dev_feature.drop('Female_Unknown', 1)
X_train_dog = X_train_feature
X_train_dog = X_train_feature.drop('AnimalType_Dog', 1)
X_dev_dog = X_dev_feature
X_dev_dog = X_dev_feature.drop('AnimalType_Dog', 1)
X_train_cat = X_train_feature
X_train_cat = X_train_feature.drop('AnimalType_Cat', 1)
X_dev_cat = X_dev_feature
X_dev_cat = X_dev_feature.drop('AnimalType_Cat', 1)